src/python/datastore/data_types.py

```diff
@@ -1108,6 +1108,12 @@ def _pre_put_hook(self):
                        fuzz_target_job_key(self.fuzz_target_name, self.job))
 
 
+class BanditProbabilities(Model):
```

Comment:
Nit: docstring

Reply:
I believe you will also need to re-run the update data types script referenced at the top of this file if you didn't already. I forgot to mention that, but the comment explains how to do this.